### PR TITLE
feat(ButtonLink): spinner support

### PR DESCRIPTION
**Pull request by marcoskolodny**

### CODE REVIEW

**Review by yceballost — APPROVED** [review]
should we include a warning in storybook about possible visual problems with spacings using loadingText in buttonLink?

File: src/__stories__/button-story.tsx
Comment (yceballost) [inline src/__stories__/button-story.tsx:1]: can you remove the white borders around the story? I really hate it xDD

image

**Comment by marcoskolodny** [comment]
> should we include a warning in storybook about possible visual problems with spacings using loadingText in buttonLink?

DIscussed offline, we decided not to do it because there is no warning for other buttons neither

File: src/__stories__/button-story.tsx
Comment (marcoskolodny) [reply src/__stories__/button-story.tsx:1]: done

**Review by atabel — APPROVED** [review]

File: src/__stories__/button-story.tsx [hunk 1]
```diff
@@ -26,21 +26,21 @@ const BackgroundTheme: StoryComponent = ({children}) => {
...
-                    style={{
-                        background: isInverse ? skinVars.colors.backgroundBrand : skinVars.colors.background,
-                    }}
-                >
+            <div
+                style={{
+                    background: isInverse ? skinVars.colors.backgroundBrand : skinVars.colors.background,
+                }}
+            >
+                <Box padding={16}>
```
[the remark this answers is not in the document]
Comment (marcoskolodny) [reply src/__stories__/button-story.tsx]: Fixed

**Pull request merged**